core/util.go

```diff
@@ -282,14 +282,17 @@ func LoadCertBundle(filename string) ([]*x509.Certificate, error) {
 func LoadCert(filename string) (cert *x509.Certificate, err error) {
```

Comment:
Good plan getting rid of the named returns below. Let's also get rid of them in the function signature.

Reply:
Good catch! Done.